tx.go

```diff
@@ -83,13 +83,13 @@ func newTransaction(ctx context.Context, c *connection, opts driver.TxOptions) (
 
 	switch sql.IsolationLevel(opts.Isolation) {
 	case sql.LevelReadUncommitted:
-		queryStr += " READ UNCOMMITTED"
+		queryStr += " ISOLATION LEVEL READ UNCOMMITTED"
```

Comment:
"ISOLATION LEVEL" is common for all isolation levels, is it better to add it to line 82?